serverless/src/app.js

```diff
@@ -45,36 +46,28 @@ export default async (event) => {
     delete body.STELLAR_NETWORK
     delete body.txFunction
 
-    // const txFunction = new Function(`
-    //   __filename,
-    //   __dirname,
-    //   require,
-    //   module,
-    //   HORIZON_URL,
-    //   STELLAR_NETWORK,
-    //   global,
-    //   globalThis,
-    //   process
-    // `, // leave [global, globalThis, process] empty to effectively unset them
-    // `
-    //   'use strict'; 
-    //   ${txFunctionCode}; 
-    //   return module.exports;
-    // `)(
-    //   __filename,
-    //   __dirname,
-    //   require,
-    //   {exports: null},
-    //   HORIZON_URL,
-    //   STELLAR_NETWORK,
-    // )
-    // const result = await txFunction(body)
+    const vm = new NodeVM({
+      console: 'off',
```

Comment:
While I agree we shouldn't let logging happen,

I think I would rather remove any CloudWatch access from the AWS IAM for the Lambda instead of removing the ability to console log, since it will likely be used during development.

Don't feel overly strongly about this, tho.

Reply:
To my knowledge it won't error, just ignore. So I'd like to keep both the console and cloudwatch off if possible. Let me do a little more testing here though.

Reply:
Yep can confirm, no errors, just no logging.

Reply:
Was able to confirm the logs are just ignored and no error happens.